mage_integrations/mage_integrations/destinations/elasticsearch/target_elasticsearch/sinks.py

```diff
@@ -185,6 +186,9 @@ def authenticated_client(self) -> elasticsearch.Elasticsearch:
             scheme = "https"
             config["ca_certs"] = self.config[SSL_CA_FILE]
 
+        if scheme == "https":
+            config['verify_certs'] = self.config[VERIFY_CERTS]
```

Comment:
I would do `self.config.get(VERIFY_CERTS, True)` here

Reply:
Effectively this looks nicer